modules/rosa-delete-dedicated-admins.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// getting_started_rosa/creating-first-rosa-cluster.adoc
+
+
+[id="rosa-delete-dedicated-admins.adoc"]
+= Revoking dedicated-admin access
+Only the user who created the cluster can revoke access for `dedicated-admin` users.
+
+.Prerequisites
+
+* You have added an Identity Provider (IDP) to your cluster.
+* You have the IDP user name for the user whose privileges you are revoking.
+* You are logged in to the cluster.
+
+.Procedure
+
+. Enter the following command to revoke access for a `dedicated-admin`:
++
+[source,terminal]
+----
+$ rosa revoke user dedicated-admin --user <idp_user_name> --cluster <cluster_name>
+----
++
+. Enter the following command to verify that your user no longer has `dedicated-admin` access. As a `dedicated-admin` you should no longer see the user in the output.
++
+[source,terminal]
+----
+$ oc get groups dedicated-admins
+----
++
+[NOTE]
+====
+A Forbidden error displays if user without `dedicated-admin` privileges runs this command.
+====
++
```

Comment:
Stray `+` here is making the "rosa-deleting-cluster" module become a subheading.

Reply:
Speaking of the "rosa-deleting-cluster" module, I know it's not part of this PR, but there's a typo in that module if you want to clean it up using this PR too.

It currently says: "To clean up your CloudFormation stack. enter the following command."


I assume that was meant to use a comma: "To clean up your CloudFormation stack, enter the following command."

Reply:
Done.